analyses/copy_number_consensus_call/run_consensus_call.sh

```diff
@@ -18,3 +18,25 @@ python3 src/scripts/merged_to_individual_files.py \
     --cnvkit ../../data/pbta-cnv-cnvkit.seg.gz \
     --freec ../../data/pbta-cnv-controlfreec.tsv.gz \
     --snake ../../scratch/config_snakemake.yaml
+
+ls ../../scratch/
+cat ../../scratch/config_snakemake.yaml
+
+## Run the Snakemake pipeline
+## -s : Point to the location of the Snakemake file
+## --configfile : Point to the location of the config file
+## -d : Specifying working directory. The "." after -d is to show the working dir is at the current folder
+## -j : Set available cores, in this case, when no number is provided, thus use all available cores
+## -p : Print shell command that will be executed
+## --restart-times : Define the times a job restarts when run into an error before giving up
+## --latency-wait: Define the number of seconds to wait for a file to show up after that file has been created
+
+snakemake \ 
```

Comment:
```suggestion
snakemake \
```

Reply:
Fixed by the commit below. https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/313/commits/d000a1b710745673b9be25ff430335d63059d037

Reply:
Thank you very much for the help! It was my mistake. I inserted a space accidentally.

Reply:
No problem! That is why we do the CI and code review!